plugins/modules/aci_l3out_floating_svi_path.py

```diff
@@ -0,0 +1,491 @@
+            }
+        }
+    }
+filter_string:
+  description: The filter string used for the request
+  returned: failure or debug
+  type: str
+  sample: ?rsp-prop-include=config-only
+method:
+  description: The HTTP method used for the request to the APIC
+  returned: failure or debug
+  type: str
+  sample: POST
+response:
+  description: The HTTP response from the APIC
+  returned: failure or debug
+  type: str
+  sample: OK (30 bytes)
+status:
+  description: The HTTP status from the APIC
+  returned: failure or debug
+  type: int
+  sample: 200
+url:
+  description: The HTTP url used for the request to the APIC
+  returned: failure or debug
+  type: str
+  sample: https://10.11.12.13/api/mo/uni/tn-production.json
+"""
+
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.cisco.aci.plugins.module_utils.aci import ACIModule, aci_argument_spec, aci_annotation_spec
+
+
+def main():
+    argument_spec = aci_argument_spec()
+    argument_spec.update(aci_annotation_spec())
+    argument_spec.update(
+        tenant=dict(type="str", aliases=["tenant_name"], required=True),
+        l3out=dict(type="str", aliases=["l3out_name"], required=True),
+        node_profile=dict(type="str", aliases=["node_profile_name", "logical_node"], required=True),
+        interface_profile=dict(type="str", aliases=["interface_profile_name", "logical_interface"], required=True),
+        state=dict(type="str", default="present", choices=["absent", "present", "query"]),
+        pod_id=dict(type="str", required=True),
+        node_id=dict(type="str", required=True),
+        encap=dict(type="str", required=True),
+        floating_ip=dict(type="str", aliases=["floating_address"]),
+        forged_transmit=dict(type="str", choices=["enabled", "disabled"], default="disabled"),
+        mac_change=dict(type="str", choices=["enabled", "disabled"], default="disabled"),
+        promiscuous_mode=dict(type="str", choices=["enabled", "disabled"], default="disabled"),
+        domain_type=dict(type="str", choices=["physical", "vmware"]),
+        domain=dict(type="str"),
+        enhanced_lag_policy=dict(type="str"),
+        access_encap=dict(type="str"),
+    )
+
+    module = AnsibleModule(
+        argument_spec=argument_spec,
+        supports_check_mode=True,
+        required_if=[
+            ["state", "present", ["domain_type", "domain", "floating_ip"]],
+            ["state", "absent", ["domain_type", "domain"]],
+        ],
+    )
+
+    tenant = module.params.get("tenant")
+    l3out = module.params.get("l3out")
+    node_profile = module.params.get("node_profile")
+    interface_profile = module.params.get("interface_profile")
+    state = module.params.get("state")
+    pod_id = module.params.get("pod_id")
+    node_id = module.params.get("node_id")
+    floating_ip = module.params.get("floating_ip")
+    encap = module.params.get("encap")
+    forged_transmit = module.params.get("forged_transmit").capitalize()
```

Comment:
What happens when .get return None?

Reply:
It will throw an error. Modified the code to fix it.